### ISSUE TITLE
3.10.20 Failing to download youtube videos.

### ISSUE THREAD

**Issue by clort81** [gpodder/gpodder#1079]
Godder Version 3.10.20
youtube-dl --version 2021.06.06
Python 3.8.5
System devuan linux

No youtubes downloading (from misesde or Ivor Cummins) 
Testing the youtube-dl works:   youtube-dl https://www.youtube.com/watch?v=tzhp2QSmOhQ

Gpodder terminal output 
[it would help if the actual URL would be displayed in the log, or visible in-client]
--------------------------------------------------------------------------------------
```

1624954472.841016 [gpodder.youtube] WARNING: Youtube get_video_info: 404 Not Found
1624954473.295826 [gpodder.download] ERROR: Download failed: No ytInitialPlayerResponse found
Traceback (most recent call last):
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/download.py", line 794, in run
    downloader = DefaultDownloader.custom_downloader(self._config, self.episode)
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/download.py", line 361, in custom_downloader
    res = registry.download_url.resolve(config, None, episode, False)
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/registry.py", line 31, in resolve
    result = resolver(item, *args)
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/youtube.py", line 186, in youtube_real_download_url
    res, duration = get_real_download_url(episode.url, allow_partial, fmt_ids)
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/youtube.py", line 214, in get_real_download_url
    raise YouTubeError('No ytInitialPlayerResponse found')
gpodder.youtube.YouTubeError: No ytInitialPlayerResponse found
1624954685.848418 [gpodder.my] WARNING: Flush requested, but sync disabled.
1624954690.383521 [gpodder.youtube] WARNING: Youtube get_video_info: 404 Not Found
1624954691.093826 [gpodder.download] ERROR: Download failed: No ytInitialPlayerResponse found
Traceback (most recent call last):
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/download.py", line 794, in run
    downloader = DefaultDownloader.custom_downloader(self._config, self.episode)
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/download.py", line 361, in custom_downloader
    res = registry.download_url.resolve(config, None, episode, False)
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/registry.py", line 31, in resolve
    result = resolver(item, *args)
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/youtube.py", line 186, in youtube_real_download_url
    res, duration = get_real_download_url(episode.url, allow_partial, fmt_ids)
  File "/media/sd/pcbak2/Projects/gpodder/src/gpodder/youtube.py", line 214, in get_real_download_url
    raise YouTubeError('No ytInitialPlayerResponse found')
gpodder.youtube.YouTubeError: No ytInitialPlayerResponse found

```
Retrying / restarting and one out of 14 downloaded  (?)

**Comment by auouymous** [comment]
Are you in Europe? If so, see #1074.

**Comment by tpikonen** [comment]
I can confirm this, but it looks like fixing this possibly requires bigger surgery in the youtube support code.

**Comment by auouymous** [comment]
Fixed in 3.10.21.
